src/main/java/com/sios/stc/coseng/run/Concurrent.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2015 SIOS Technology Corp. All rights reserved.
+ * This file is part of COSENG (Concurrent Selenium TestNG Runner).
+ * 
+ * COSENG is free software: you can redistribute it and/or modify
```

Comment:
Not aware of any former or current strategy on open source licensing for SIOS Tech.

Apache v2 is fine by me - Selenium is Apache v2 as is TestNG; which are more than key components of this project. If either were GPLv3 though we'd not be able to do Apache for this project.

http://www.apache.org/licenses/GPL-compatibility.html "Apache 2 software can therefore be included in GPLv3 projects, because the GPLv3 license accepts our software into GPLv3 works. However, GPLv3 software cannot be included in Apache projects."

Just went with GPLv3 as that was what I'd done FVORGE as...

I'll re-adjust to Apachev2